infomaniak-build-tools/kDrive-template.html

```diff
@@ -19,18 +19,28 @@
 <!DOCTYPE html>
 <html>
 <body>
-<h1>Release notes - kDrive 3.6.6</h1>
+<h1>Release notes - kDrive 3.6.7</h1>
+<h2><b>New features</b></h2>
+  <ul>
+    <li>Change the update mechanism to allow progressive delivery of new version</li>
+  </ul>
 <h2><b>Bug fixes</b></h2>
   <ul>
-    <li>Fixed a sync issue occurring on files whose name contains a special character</li>
-    <li>Corrected behaviour in the event of disconnection</li>
-    <li>Fixed issue where an empty file list is retrieved in case of poor network connection</li>
-    <li>Fixed an issue causing many conflicting files when using specific software</li>
-    <li>Fixed a sync issue generating infinite loop</li>
-    <li>Windows - Code optimization to reduce duration of steps 1 and 2</li>
-    <li>macOS - Fixed unwanted popup message on session logout</li>
-    <li>Fixed several crashes related to logging</li>
-    <li>Fixed several crashes related to sync pausing</li>
+    <li>Fixed new file wrongly downloaded in LiteSync mode</li>
+    <li>Fixed missing share links entries in contextual menu</li>
+    <li>Consolidation of error management during the propagation step</li>
+    <li>Do not block application startup if local sync folder is missing</li>
+    <li>Add "*.tmp" files to default exclusion templates</li>
```

Comment:
Add => Added

Reply:
Done in ee84a9ed3e8c9765067d576481598f818eb5669c